test/e2e/storage/drivers/csi.go

```diff
@@ -139,17 +139,16 @@ var _ storageframework.EphemeralTestDriver = &hostpathCSIDriver{}
 // InitHostPathCSIDriver returns hostpathCSIDriver that implements TestDriver interface
 func InitHostPathCSIDriver() storageframework.TestDriver {
 	capabilities := map[storageframework.Capability]bool{
-		storageframework.CapPersistence:         true,
-		storageframework.CapSnapshotDataSource:  true,
-		storageframework.CapMultiPODs:           true,
-		storageframework.CapBlock:               true,
-		storageframework.CapPVCDataSource:       true,
-		storageframework.CapControllerExpansion: true,
-		storageframework.CapOfflineExpansion:    true,
-		storageframework.CapOnlineExpansion:     true,
-		storageframework.CapSingleNodeVolume:    true,
-		storageframework.CapReadWriteOncePod:    true,
```

Comment:
Is dropping `CapReadWriteOncePod` and `CapMultiplePVsSameID` intentional?

Reply:
Nope, should be fixed now. Thank you!